components/utils/Profile.tsx

```diff
@@ -0,0 +1,24 @@
+import Avatar from 'components/Avatar'
+
+export default function Profile({
+  avatar,
+  name,
+  description,
+  children,
+}: {
+  avatar?: string
+  name: string
+  description?: string
+  children?: React.ReactNode
+}) {
+  return (
+    <div className="flex w-full flex-col items-center p-4">
+      <Avatar avatar={avatar} />
+      <div className="my-5 flex flex-col gap-4">
+        <span className="text-xl font-bold text-white">{name}</span>
+        <span className="text-xl font-bold text-white">{description}</span>
```

Comment:
You have marked the `description` prop as optional by using `?`, you are correct it should be optional. However you are _always_ rendering a `span` for it here. Instead we should only render this `span` if `description` has been set:

```jsx
{description && <span className="text-xl font-bold text-white">{description}</span>}
```